applications/serial_lte_modem/src/slm_at_tcpip.c

```diff
@@ -466,25 +466,26 @@ static int do_listen(void)
 
 static int do_accept(void)
 {
-	int ret;
+	int fd;
 	char peer_addr[INET_ADDRSTRLEN];
 	socklen_t len = sizeof(struct sockaddr_in);
 
-	ret = accept(client.sock, (struct sockaddr *)&remote, &len);
-	if (ret < 0) {
-		LOG_ERR("accept() failed: %d/%d", -errno, ret);
+	fd = accept(client.sock, (struct sockaddr *)&remote, &len);
+	if (fd == -1) {
+		LOG_ERR("accept() failed: %d", -errno);
```

Comment:
How about use INVALID_SOCKET instead of -1

Reply:
Prefer to keep current code.
-1 is official BSD accept() return when API fails.
INVALID_SOCKET is SLM definition and happens to be -1.